hw03_phillipsj2.py

```diff
@@ -0,0 +1,209 @@
+######################################################################
+# Authors: Chase Phillips
+# Username: phillipsj2
+#
+# Assignment: HW03
+#
+# Purpose: A program that uses turtles to draw a complex object. This is also an intro to Git's more complex nature
+#
+#https://docs.google.com/document/d/1ScrNyDoxQ187_iHzXMwE0PJ0ZUmyG6eX1xUxiFIGk-Y/edit?usp=sharing
+######################################################################
+
+import turtle
+
+def draw_trailer(rect, door, w1, w2):
+    """This is meant to draw the trailer, a mobile home. Compeleted with doors and windows"""
+    #This is going to draw a blue trailer shape + the grey roof
+    rect.fillcolor('blue')
+    rect.begin_fill()
+    for i in range(2):
+        rect.forward(300)
+        rect.left(90)
+        rect.forward(125)
+        rect.left(90)
+    rect.end_fill()
+
+    #This is drawing the "roof" of the trailer to add some depth
```

Comment:
You don't need to fix this:

These two functions got large. Breaking down the problem further might have identified functions for each of these commented sections (roof, door, etc.).